.github/workflows/deploy-stage.yml

```diff
@@ -1,9 +1,9 @@
-name: Deploy Console Kit (STAGE)
+name: Deploy Console Kit (PRODUCTION)
 
 on:
   push:
     branches:
-      - dev
+      - main
```

Comment:
looks like this branch need to be changed to dev to match the deploy.stage.yml name in this file.